controller/instance_manager_controller.go

```diff
@@ -272,11 +276,14 @@ func (imc *InstanceManagerController) syncInstanceManager(key string) (err error
 		return err
 	}
 
+	log := imc.logger.WithField("instance manager", im.Name)
```

Comment:
you can use `getLoggerForInstanceManager`

Reply:
Thank you. Fixed